### PR TITLE
main: Default package name to current directory

### PR DESCRIPTION
**Pull request by carolynvs**
When running tinygo build/run/test, and the positional argument for the package is not specified, default it to "."

The go command for run doesn't default the package, but it seems like it could, and it is a nice feature, so I did it for tinygo run as well.

### CODE REVIEW

**Comment by deadprogram** [comment]
Agreed with doing this for `tinygo test` because that is the same thing that `go test` does.

However, `go run` requires an argument of what to run, so we should probably stick with that for `tinygo run`.

What do you think?

**Comment by carolynvs** [comment]
Did you mean that this should only add `.` as the default for build but not run?

**Comment by m-chichikalov** [comment]
Also noticed that tinygo build requered a path and does not build from PWD. 👍

**Comment by aykevl** [comment]
I agree with @carolynvs, it would be nice if `tinygo run` would just work (using the current directory if no path is specified). I don't see a reason why it shouldn't.

File: main.go [hunk 1]
```diff
@@ -634,16 +634,15 @@ func main() {
 			usage()
 			os.Exit(1)
 		}
-		if flag.NArg() != 1 {
-			fmt.Fprintln(os.Stderr, "No package specified.")
-			usage()
-			os.Exit(1)
+		pkgName := "."
+		if flag.NArg() == 1 {
```
Comment (aykevl) [inline main.go]: So when you pass more than one parameter, it still defaults to the current directory? I think we should print an error when there is more than one positional parameter specified. (The same goes for the other two commands below).

**Comment by aykevl** [comment]
Actually, I see why `go run` requires an explicit package path. Because if you want to pass command line arguments (and you do, for most programs), then those are taken from the `go run` command line. For example:

```go
package main

import "fmt"
import "os"

func main() {
	fmt.Println("args:", os.Args)
}
```

```
$ go run . asdf
args: [/tmp/go-build046346693/b001/exe/hello asdf]
```

**Comment by deadprogram** [comment]
So seems like just remove this from `run`, but leave in the change for both `build` and `test`. This lets us retain the same syntax as that used by the main Go implementation.

**Comment by carolynvs** [comment]
Gotcha, I've ammended the commit to just make the change for build and test.

File: main.go (hunk 1 above)
Comment (deadprogram) [reply main.go]: That seems like a good idea.

**Comment by carolynvs** [comment]
I've fixed build/test to fail when more than one positional argument is passed. I also noticed that test wasn't in the helptext so I snuck that in quick. 😀

**Comment by deadprogram** [comment]
Thanks for this PR @carolynvs and also for accommodating the requested changes. Now merging.

**Pull request merged**